src/platforms/javascript/guides/cordova/upload-debug.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Debug Information
+sidebar_order: 2
+description: "Learn more about uploading debug symbols for when building your app."
```

Comment:
@jennmueng what's about source maps? does it work automatically for Android and iOS?

Reply:
Yes it does, it actually automatically uploads **js**-only sourcemaps on at build time on `cordova build`. Maybe we can create a Cordova-specific source maps section @PeloWriter as the current sourcemaps section for JS is geared towards browser and mostly not relevant to Cordova.